apps/rabbitmq_prelaunch/src/rabbit_prelaunch_sighandler.erl

```diff
@@ -0,0 +1,85 @@
+-module(rabbit_prelaunch_sighandler).
+-behaviour(gen_event).
+
+-export([setup/0,
+         init/1,
+         handle_event/2,
+         handle_call/2,
+         handle_info/2,
+         terminate/2,
+         code_change/3]).
+
+%% CAUTION: Signal handling in this module must be kept consistent
+%% with the same handling in rabbitmq-server(8).
+
+%% #{signal => default | ignore | stop}.
+-define(SIGNALS,
```

Comment:
We should rename this to `SIGNALS_HANDLED_BY_US`, so that it's on par with `SIGNAL_HANDLED_BY_ERLANG`.